packages/math-input/src/store/keypad-reducer.ts

```diff
@@ -1,13 +1,13 @@
 import KeyConfigs from "../data/key-configs";
-import Keys from "../data/keys";
+import Key from "../data/keys";
 
 import {defaultKeypadType} from "./shared";
 
 import type {Action} from "./actions";
 import type {KeypadState} from "./types";
 
 const initialKeypadState = {
-    extraKeys: ["x", "y", Keys.THETA, Keys.PI],
+    extraKeys: ["x", "y", "THETA", "PI"] as Key[],
```

Comment:
I'm still ramping up on Typescript, but I think this could be `as const` which avoids us [asserting](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions).